projects/experimental/components/badge/badge.style.less

```diff
@@ -0,0 +1,461 @@
+
+        &._hoverable:hover {
+            background-color: var(--tui-secondary-hover);
+        }
+
+        &._hoverable:active {
+            background-color: var(--tui-secondary-active);
+        }
+
+        &[data-platform='ios'],
+        &[data-platform='android'] {
+            background-color: var(--tui-base-06);
+        }
+    }
+
+    &[data-appearance='error'] {
+        --left-content-color: var(--tui-error-fill);
+        background-color: var(--tui-error-bg);
+        color: var(--tui-text-01);
+
+        &._hoverable:hover,
+        &._hoverable:active {
+            background-color: var(--tui-error-bg-hover);
+        }
+
+        &[data-mode='onDark'] {
+            background-color: var(--tui-error-bg-night);
+            color: var(--tui-text-01-night);
+
+            &._hoverable:hover,
+            &._hoverable:active {
+                background-color: var(--tui-error-bg-night-hover);
+            }
+        }
+    }
+
+    &[data-appearance='success'] {
+        --left-content-color: var(--tui-success-fill);
+        background-color: var(--tui-success-bg);
+        color: var(--tui-text-01);
+
+        &._hoverable:hover,
+        &._hoverable:active {
+            background-color: var(--tui-success-bg-hover);
+        }
+
+        &[data-mode='onDark'] {
+            background-color: var(--tui-success-bg-night);
+            color: var(--tui-text-01-night);
+
+            &._hoverable:hover,
+            &._hoverable:active {
+                background-color: var(--tui-success-bg-night-hover);
+            }
+        }
+    }
+
+    &[data-appearance='warning'] {
+        --left-content-color: var(--tui-warning-fill);
+        background-color: var(--tui-warning-bg);
+        color: var(--tui-text-01);
+
+        &._hoverable:hover,
+        &._hoverable:active {
+            background-color: var(--tui-warning-bg-hover);
+        }
+
+        &[data-mode='onDark'] {
+            background-color: var(--tui-warning-bg-night);
+            color: var(--tui-text-01-night);
+
+            &._hoverable:hover,
+            &._hoverable:active {
+                background-color: var(--tui-warning-bg-night-hover);
+            }
+        }
+    }
+
+    &[data-appearance='info'] {
+        --left-content-color: var(--tui-info-fill);
+        background-color: var(--tui-info-bg);
+        color: var(--tui-text-01);
+
+        &._hoverable:hover,
+        &._hoverable:active {
+            background-color: var(--tui-info-bg-hover);
+        }
+
+        &[data-mode='onDark'] {
+            background-color: var(--tui-info-bg-night);
+            color: var(--tui-text-01-night);
+
+            &._hoverable:hover,
+            &._hoverable:active {
+                background-color: var(--tui-info-bg-night-hover);
+            }
+        }
+    }
+
+    &[data-appearance='neutral'] {
+        --left-content-color: var(--tui-neutral-fill);
+        background-color: var(--tui-neutral-bg);
```

Comment:
nit: If we replace all `background-color` with `background` we'll save a few bytes :)